### ISSUE TITLE
Documentation for multiple versions

### ISSUE THREAD

**Issue by dpole** [litebird/litebird_sim#49]
The `litebrid_sim` github page currently displays only the documentation of the master branch.
This has several limitation

* If, for any reason, a user wants to use a previous version he does not have a documentation for that
* We can not share documentation for among developers about developing branches
* We can not preview how the documentation will render exactly until we merge with the master branch

Possible decision
1. We accept these constraints
2. We use readthedocs
3. We keep using gihub pages but point the website to the `gh-pages` branch instead of the `docs` folder of the `master` branch. Then each branch have to write its documentation to a folder named after the branch in `gh-pages`

**Comment by ziotom78** [comment]
I dislike the current state of things, so please drop out option 1. Regarding option 3, I think it's more burdensome to set up and maintain. 

Finally, I would *love* to use Readthedocs! When I set up the current configuration, this repository was still private and Readthedocs was thus ruled out. But there's no reason to avoid it now that the repository is public.

TLDR: I strongly favour option 2.

**Comment by dpole** [comment]
rtd is setup, see #51 
https://litebird-sim.readthedocs.io/en/latest/
